docs/book/application-integration/usage-in-a-laminas-mvc-application.md

```diff
@@ -0,0 +1,181 @@
+# Usage in a laminas-mvc Application
+
+The following example shows _one_ potential use case of laminas-form within
+a laminas-mvc based application. The example uses a module, a controller and the
+laminas-form element manager.
+
+The example is based on the [tutorial application](https://docs.laminas.dev/tutorials/getting-started/overview/)
+which builds an album inventory system.
+
+Before starting, make sure laminas-form is installed and configured.
+
+## Create Form
+
+Create a form as separate class, e.g. `module/Album/src/Form/AlbumForm.php`:
+
+```php
+namespace Album\Form;
+
+use Laminas\Form\Element\Text;
+use Laminas\Form\Form;
+
+class AlbumForm extends Form
+{
+    public function init()
+    {
+        // Title
+        $this->add(
+            [
+                'name'    => 'title',
+                'type'    => Text:class,
+                'options' => [
+                    'label' => 'Title',
+                ],
+            ]
+        );
+    
+        // …
+    }
+}
+```
+
+## Using Form
+
+### Create Controller
+
+Using the form in a controller, e.g.
+`module/Album/Controller/AlbumController.php`:
+
+```php
+namespace Album\Controller;
+
+use Album\Form\AlbumForm;
+use Laminas\Form\FormInterface;
+use Laminas\Mvc\Controller\AbstractActionController;
+
+class AlbumController extends AbstractActionController
+{
+    /** @var FormInterface */
+    private $form;
+    
+    public function __construct(FormInterface $form)
```

Comment:
I'd prefer to have `AlbumForm` here, then we will be able to take advantages of zend-di and autowiring.

(yeah, with interface it will be still possible but additional configuration will be required)

Reply:
Good point!